lib/ui/painting.dart

```diff
@@ -1741,6 +1741,16 @@ class ImageFilter extends NativeFieldWrapperClass2 {
     _initBlur(sigmaX, sigmaY);
   }
   void _initBlur(double sigmaX, double sigmaY) native 'ImageFilter_initBlur';
+
+  /// Creates an image filter that applies a matrix transformation.
+  ImageFilter.matrix(Float64List matrix4,
```

Comment:
can filterQuality be null / have a default?

Reply:
Added default (SkFilterQuality.low)